lib/std/Thread.zig

```diff
@@ -695,7 +695,7 @@ const PosixThreadImpl = struct {
         defer assert(c.pthread_attr_destroy(&attr) == .SUCCESS);
 
         // Use the same set of parameters used by the libc-less impl.
-        const stack_size = @max(config.stack_size, c.PTHREAD_STACK_MIN);
```

Comment:
`PTHREAD_STACK_MIN` usage make sense. `pthread_attr_setstacksize()` would return `EINVAL` if the value passed is less than `PTHREAD_STACK_MIN`

Reply:
Unfortunately the values used for most of the operating systems are not verified, so this commit will remain reverted. In practice, 100% of the time `config.stack_size` will be orders of magnitude larger than the pthread stack minimum size.